src/app/components/activity/feed/event-item/event-item.vue

```diff
@@ -152,7 +153,13 @@
 				/>
 			</div>
 
-			<app-event-item-controls class="-controls" :post="post" :video="video" @expand="onExpand()" />
+			<app-event-item-controls
+				class="-controls"
+				:post="post"
+				:video="video"
+				:show-user-follow-on-like="feed.shouldShowFollowOnLike"
```

Comment:
shorter attr name would be appreciated

Reply:
Any less specific and its too ambiguous. if I don't say `show` it means it'll auto follow. If I don't say `User`, it'll be confused with following the resource the event item was posted for. e.g. the game a post was made on. if I remove `OnLike` you might assume it always shows, or you wouldn't know which control would show the follow widget. Hmm..

I guess `OnLike` is the most specific bit i can omit, so `showUserFollow`?

Reply:
yeah, I liked "show-user-follow" since the controls can figure out how it wants to do that